showcase/src/data/data.js

```diff
@@ -0,0 +1,31 @@
+export default [
+  {
+    image: 'http://via.placeholder.com/180x320',
+    username: 'test user',
+    tagline: 'Came for the code, stayed for the code',
+    bio: 'I am a human being. No really, I am totally not a computer',
+    projects: [
+      'project1',
+      'project2',
+      'project3',
+      'project4',
+      'project5',
+      'project6',
+    ],
+  },
+  {
+    image: 'http://via.placeholder.com/180x320',
+    username: 'test user 2',
+    tagline: 'Came for the pizza',
+    bio: 'I am a human being. No really, I am totally not a computer',
+    projects: ['project1', 'project2', 'project3', 'project4'],
```

Comment:
This is good for now, but of course, projects will need to be their own structure. I'm curious/concerned how large each individual's json data structure will need to be. We are at 30 lines with about half the data we'll eventually need. Maybe each user can create their own file by copying a template and export that variable and include it in this structure? They'd have to edit two files but their edit in the shared file would be smaller. If we need to change the data structure in the future we'll have to have a script read it in and create new files with the changes.